proposed/clock-meta.md

```diff
@@ -41,17 +39,36 @@ calling `time()` or `date()`.
   described in this document, so it is not a coding standard;
 * This PSR does not provide a recommendation on how to handle timezones when 
   retrieving the current time. This is left up to the implementation.
+* This PSR does not handle any scheduling methods like `sleep()` or `wait()` because such methods are not related to retrieving the current time.
 
 ## 4. Approaches
 
 ### 4.1 Chosen Approach
 
-We have decided to formalize the existing practices, used by several other packages
-out in the wild. Some of the popular packages providing this functionality are: 
-`lcobucci/clock`, `kreait/clock`, `ergebnis/clock`, and `mangoweb/clock`. Some providing
+We have decided to formalize the existing practices, used by several other packages out in the wild. Some of the popular packages providing this functionality are: 
+
+* [`lcobucci/clock`](https://packagist.org/packages/lcobucci/clock)
+* [`kreait/clock`](https://packagist.org/packages/kreait/clock)
+* [`ergebnis/clock`](https://packagist.org/packages/ergebnis/clock)
+* [`mangoweb/clock`](https://packagist.org/packages/mangoweb/clock)
+
+(This list is not exhaustive!)
+
 interfaces, and some relying on overloading (extending) the Clock class to mock the
 current time.
 
+These implementations all provide a `now()` method which returns a `DateTimeImmutable` object. As the `DateTimeImmutable` object allows retrieving the Unix timestamp, by calling `getTimestamp()` or `format('u.U')`, this interface does not define any special methods to retrieve a Unix timestamp or any other time information that is not available from a `DateTimeImmutable` object. 
+
+### 4.2 Timezones
+
+Time by now is defined by interaction of electromagnetic radiation with the excited states of certain atoms where the SI defines one second as the duration of 9192631770 cycles of radiation corresponding to the transition between two energy levels of the ground state of the caesium-133 atom at 0K. This means that retrieving the current time will always return the same time, no matter where it is observed. While the timezone defines *where* the time was observed it does not modify the actual "slice" of time.
+
+This means that for the sake of this PSR the timezone is considered an implementation detail of the interface. 
+
+It is up to the implementation to make sure that the timezone is handled according to the business logic of the application. That is either by making sure that a call to `now()` will only return a `DateTimeImmutable` object with a known timezone (implicit contract) or by explicitly changing the timezone to be correct for the application. This an be done by calling `setTimezone()` to create a new `DateTimeImmutable` object with the given timezone. 
```

Comment:
Yes x 1000! This paragraph is the most important to me. I hope the one above about caesium-133 atom won't loose readers before they reach this 🤣

Reply:
Good catch! Thanks for spotting!